controllers/workspace/devworkspace_controller_test.go

```diff
@@ -653,20 +650,23 @@ var _ = Describe("DevWorkspace Controller", func() {
 		It("Sets the runtimeClassName from the attribute instead of the DWOC", func() {
 			By("Creating DWOC with a runtimeClassName")
 			runtimeClassName := "test-runtime-class"
-			dwoc := generateDwoc("devworkspace-operator-config")
-			dwoc.Config = &controllerv1alpha1.OperatorConfiguration{
+			config.SetGlobalConfigForTesting(&controllerv1alpha1.OperatorConfiguration{
 				Workspace: &controllerv1alpha1.WorkspaceConfig{
 					RuntimeClassName: &runtimeClassName,
 				},
-			}
+			})
+			defer config.SetGlobalConfigForTesting(nil)
 
-			createObject(dwoc)
-			defer deleteObject(dwoc)
+			devworkspace := &dw.DevWorkspace{}
+			createDevWorkspace(devWorkspaceName, "test-devworkspace.yaml")
+
+			By("Adding runtime-class annotation to the DevWorkspace")
+			Eventually(func() error {
+				devworkspace = getExistingDevWorkspace(devWorkspaceName)
+				devworkspace.Spec.Template.Attributes.PutString(constants.RuntimeClassNameAttribute, "test-runtime-class-attribute")
+				return k8sClient.Update(ctx, devworkspace)
+			}, timeout, interval).Should(Succeed(), "DevWorkspace should get `runtime-class: test-runtime-class-attribute` annotation")
```

Comment:
Since we're adding an attribute, should we make the error message say: "DevWorkspace should get `runtime-class: test-runtime-class-attribute` attribute" (instead of annotation)?